site/src/pages/CreateUserPage/CreateUserForm.tsx

```diff
@@ -20,6 +21,8 @@ export const Language = {
   emailLabel: "Email",
   passwordLabel: "Password",
   usernameLabel: "Username",
+  nameLabel: "Full name",
```

Comment:
Nit: I would use "Name" instead of "Full name".

Reply:
Again, I would much prefer to use "full name" in the UI/CLI to distinguish it from "username". The API refers to it as `name` however.